design-documents/atomics-ABI.rst

```diff
@@ -0,0 +1,308 @@
+..
+   Copyright (c) 2023, Arm Limited and its affiliates.  All rights reserved.
+   CC-BY-SA-4.0 AND Apache-Patent-License
+   See LICENSE file for details
+
+.. _ARMARM: https://developer.arm.com/documentation/ddi0487/latest
+.. _PAPER: https://doi.org/10.1109/CGO57630.2024.10444836
+
+Rationale Document for C11 Atomics ABI.
+***************************************
+
+Preamble
+========
+
```

Comment:
Could you add a Scope section like the BuildAttributes design https://github.com/ARM-software/abi-aa/pull/230/files#diff-f662c6d976647c3fea89e33e36da10a2d57957cb9261b76512a4b35ef0fa3686R157

Just want to capture that this is rationale and not requirements and any ambiguity is resolved in favour of the specfication.